### PR TITLE
Check for dd-agent user before creating it

### PR DESCRIPTION
**Pull request by talwai**
Addresses https://github.com/DataDog/dd-agent/issues/1587

### CODE REVIEW

File: package-scripts/datadog-agent/postinst [hunk 1]
```diff
@@ -39,8 +39,13 @@ if [ "$DISTRIBUTION" != "Darwin" ]; then
           configure)
               update-rc.d datadog-agent defaults
               update-rc.d datadog-agent enable >/dev/null 2>&1
-              adduser --system dd-agent --disabled-login --shell /bin/sh --no-create-home --quiet
-              usermod -d /opt/datadog-agent dd-agent
+              # Only add dd-agent user if it doesn't already exist
+              id -u dd-agent &>/dev/null
```
Comment (elafarge) [inline package-scripts/datadog-agent/postinst]: If you really want that line to ouput nothing you should change that to `id -u dd-agent >/dev/null 2>&1` :)

File: package-scripts/datadog-agent/postinst [hunk 2]
```diff
@@ -39,8 +39,13 @@ if [ "$DISTRIBUTION" != "Darwin" ]; then
...
+              USER_EXISTS=$?
+              if [ ! $USER_EXISTS -eq 0 ]; then
+                  adduser --system dd-agent --disabled-login --shell /bin/sh --no-create-home --quiet
```
Comment (elafarge) [inline package-scripts/datadog-agent/postinst]: I know there's not that many echoes in our `postinst` script but I think the creation of a new user on the machine is worth notifying. Could you just add a little something like `echo "Creating user 'dd-agent'."`  ?

**Comment by elafarge** [comment]
I tested that on both `dash` (Ubuntu) and `bash` shells, it works like a charm. Good job !

I just had two little observations about silencing the output of one command and notifying our customers that we are actually creating a new user.

After that, let's just :ship: it !

**Comment by talwai** [comment]
@elafarge Thanks for the review. Implemented your suggestions, and merging now

**Pull request merged**